onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -2935,11 +2945,12 @@ def aten_margin_ranking_loss(
 
     raise NotImplementedError()
 
-
+@torch_op("aten::masked_fill")
 def aten_masked_fill(self: TensorType, mask: TensorType, value: TensorType) -> TensorType:
```

Comment:
I would update the types like above here and for all new functions. LMK if you needed more info.

Reply:
I see. I will try to replace annotations

Reply:
Done